Tweaks so pcalc will build with flex 2.5.4a and 2.5.31.

Patch by Mike Frysinger <vapier@gentoo.org>

--- pcalcl.l
+++ pcalcl.l
@@ -15,8 +15,6 @@
 extern YYSTYPE yylval;
 extern  int lineno;
 
-/* default yywrap function - always treat EOF as an EOF */
-#define yywrap() 1
 
 
 char    *name;
--- Makefile
+++ Makefile
@@ -36,5 +36,5 @@
 
 pcalc:  $(OB0) $(OB1) $(OB2) $(OB3) $(OB4) $(OB5) $(OB6) $(OB7) $(OB8) $(OB9)
-	$(CC) $(CFLAGS) $(OB0) $(OB1) $(OB2) $(OB3) $(OB4) $(OB5) $(OB6) $(OB7) $(OB8) $(OB9) -o pcalc -lm
+	$(CC) $(CFLAGS) $(OB0) $(OB1) $(OB2) $(OB3) $(OB4) $(OB5) $(OB6) $(OB7) $(OB8) $(OB9) -o pcalc -lm -lfl
 
 pcalc.c: pcalc.y
